docs/release-notes/2.27.0.md

```diff
@@ -0,0 +1,143 @@
+
+- [PMM-9542](https://jira.percona.com/browse/PMM-9542): PMM now predefines certain flags that allow users to set all other [VictoriaMetrics parameters](https://docs.victoriametrics.com/#list-of-command-line-flags) as environment variables. For more information see [Documentation](https://www.percona.com/doc/percona-monitoring-and-management/2.x/details/victoria-metrics.html#environment-variables).
+
+- [PMM-9452](https://jira.percona.com/browse/PMM-9452): With this release of PMM, we have implemented a simplified password change method for the default admin user using the command line parameter change-admin-password.
+
+- [PMM-9339](https://jira.percona.com/browse/PMM-9339): The Use TLS option in webhook settings has been renamed to Show TLS setting to better reflect its functionality.
+ 
+- [PMM-9319](https://jira.percona.com/browse/PMM-9319): Synchronized Platform and PMM roles: We have updated PMM permissions to ensure that Administrators of Percona Portal organizations are also granted Admin role in PMM. 
+
+- [PMM-9182](https://jira.percona.com/browse/PMM-9182): Integrated Alerting: Added Silence All  option for when you want to stop notifications from all alerting rules at once.
+
+- [PMM-9164](https://jira.percona.com/browse/PMM-9182): Integrated Alerting: You can now use an existing rule as a source for new ones instead of using a template.
+ 
+- [PMM-9148](https://jira.percona.com/browse/PMM-9148): You can now share a link for Query Analytics at the click of a button with the *Copy Link*.
+ 
+ - [PMM-8794](https://jira.percona.com/browse/PMM-8794): For consistency, we have implemented a unified 24 hours time format for backup management.
+ 
+- [PMM-8045](https://jira.percona.com/browse/PMM-8045): DBaaS: With this version of PMM, we have added a warning about the deletion of API keys so that the user is forewarned before deleting the API key.
+  
+- [PMM-4466](https://jira.percona.com/browse/PMM-4466): With this release of PMM we have migrated Grafana from using SQLite to PostgreSQL datasource for better performance of Auth sessions.
+
+## Bugs Fixed
+
+ - [PMM-9797](https://jira.percona.com/browse/PMM-9797): Fixed an issue where the data on the Home Dashboard was represented incorrectly on the Stat Panel, which could potentially confuse the user.
+ 
+ - [PMM-9783](https://jira.percona.com/browse/PMM-9783): Fixed an issue where QAN failed to work after an upgrade.
+ 
+ - [PMM-9757](https://jira.percona.com/browse/PMM-9757): Fixed an issue where the metrics for MongoDB were not being exposed when the database connection was getting lost.
+ 
+ - [PMM-9692](https://jira.percona.com/browse/PMM-9692): Ensured that Technical preview disclaimer message is always displayed in the Percona Platform section in the PMM Settings. 
+ 
+- [PMM-9671](https://jira.percona.com/browse/PMM-9671): Fixed an issue where the PMM configuration was lost due to pmm-managed crashing when the PMM agent version was null.
```

Comment:
Here we need a real user explanation. 
Our users don't care/know about  internal components like pmm-amanged

Reply:
@puneet0191 Could you please provide inputs on this...

Reply:
this issue could be seen by user as well, it affects PMM. From user perspective it is defined in a ticket description. Title could look something like: Update to 2.26.0 failed, PMM is crashing.

Reply:
@denisok How about this?

Fixed an issue where an upgrade to 2.26.0 failed as PMM crashed.